setup.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup, find_packages
+import re, ast
+
+with open('requirements.txt') as f:
+	install_requires = f.read().strip().split('\n')
+
+# get version from __version__ variable in erpnext_template/__init__.py
+_version_re = re.compile(r'__version__\s+=\s+(.*)')
+
+with open('erpnext_template/__init__.py', 'rb') as f:
+	version = str(ast.literal_eval(_version_re.search(
+		f.read().decode('utf-8')).group(1)))
+
+setup(
+	name='erpnext_template',
+	version=version,
+	description='Description',
+	author='Publisher',
+	author_email='template@email.com',
```

Comment:
Update those to have the same values (or markers) as the rest of the application.